build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.4.2'
+    classpath 'com.android.tools.build:gradle:3.5.0'
```

Comment:
Is there a specific reason you're updating the Gradle version? Prefer not making that change with other code changes in case there are downstream implications.

Reply:
Done. Accidental change